src/js/components/Button/stories/IconLabelCustom.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { Add } from 'grommet-icons';
+
+import { Box, Button, Grommet } from 'grommet';
+
+const customIconLabel = {
+  button: {
+    color: 'pink',
+  },
+};
+
+const CustomIconLabel = () => (
```

Comment:
Thank you for adding a story! Since Button component is pretty busy with stories, do you think we can combine this story into the existing Button/stories/Colored.js story?

Reply:
I can do this!

Reply:
we can take care of this later. the PR is open for so long.